data/ecosystems/c/cosmos.toml

```diff
@@ -103,7 +103,7 @@ sub_ecosystems = [
   "Provenance",
   "Provenance Blockchain",
   "Pylons",
-  "quasar finance",
+  "Quasar Finance",
```

Comment:
The title should also be changed inside the quasar-finance.toml file. Names need to match, otherwise the PR validator will fail, as it is failing now.